frodo-kem/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+authors = ["Michael Lodder <redmike7@gmail.com>"]
+categories = ["cryptography"]
+description = "A pure rust implementation of FrodoKEM that also includes some experimental features for Zero-Knowledge Proof of Encrypted Correctness"
+documentation = "https://docs.rs/frodo-kem-rs"
+edition = "2021"
+homepage = "https://github.com/mikelodder7/frodoKem"
+keywords = ["quantum", "post", "zero", "knowledge", "encryption"]
+license = "Apache-2.0 OR MIT"
+name = "frodo-kem-rs"
+readme = "README.md"
+repository = "https://github.com/mikelodder7/frodoKem"
+version = "0.2.0"
+
+[features]
+default = ["frodo640aes", "frodo640shake", "frodo976aes", "frodo976shake", "frodo1344aes", "frodo1344shake", "serde"]
+hazmat = []
+experimental = []
+frodo640aes = ["aes"]
+frodo640shake = []
+frodo976aes = ["aes"]
+frodo976shake = []
+frodo1344aes = ["aes"]
+frodo1344shake = []
+serde = ["hex", "dep:serde"]
+
+[dependencies]
+aes = { version = "0.8", optional = true }
+hex = { version = "0.4", optional = true }
+rand_core = { version = "0.6", features = ["std"] }
```

Comment:
Is the std feature needed? As this also pulls in the `getrandom` crate for `OsRng` support.